addons/html_editor/static/src/fields/html_field.js

```diff
@@ -67,6 +74,10 @@ export class HtmlField extends Component {
                 );
             }
         });
+
+        this.dynamicPlaceholder = this.props.dynamicPlaceholder
```

Comment:
That is complicated to have to configure a placeholder outside the plugin placeholder. Everything should be contained within the plugin.

That is also going to be easier for the unit tests.

Reply:
This is a limitation of the current `useDynamicPlaceholder()` wich need a reference to an OWL `Component` to properly function.
Changing this would required to refactor of the way  `useDynamicPlaceholder()`works. wich is out of the scope of this. IMO.